cmd/podman/varlink.go

```diff
@@ -62,7 +63,13 @@ func varlinkCmd(c *cli.Context) error {
 
 	// Run the varlink server at the given address
 	if err = service.Listen(args[0], timeout); err != nil {
-		return errors.Errorf("unable to start varlink service")
+		switch err.(type) {
+		case varlink.ServiceTimeoutError:
+			fmt.Printf("varlink service timed out (use --timeout to change)\n")
```

Comment:
I think the hard part is to decide what we want to print. @baude, @rhatdan What do you think?

Reply:
Maybe:  "varlink service timed out (use --timeout to increase session time beyond one second)\n", but yeah @vrothberg it's hard to be concise here and get the point across.  I'm not real hip with my suggestion either.

But regardless of this change, does it make sense to have the varlink --timeout flag work in mililseconds?  Can we convert that to seconds?  I think most people would assume it would be seconds not milliseconds, easier to enter into the command line too.

Reply:
Thanks a lot, @TomSweeneyRedHat, for the feedback. I will wait for more feedback but I already prefer you suggestion above!

Reply:
you could also say varlink service expired ... and then keep the the (use --timeout ...)

my only deal is whether it should be logrus or actually a print

Reply:
@baude If you feel more comfortable with logrus, I understand that. We can still change that at a later point if users are complaining.

To conclude:
- We go for logrus info
- "varlink service expired (use --timeout to increase session time beyond one second)

Any objections?

Reply:
I updated the commit.